test/performance/long-running-tests/README.md

```diff
@@ -0,0 +1,174 @@
+
+   ```bash 
+   export CL2_DIR=$HOME/perf-tests/clusterloader2
+   ```
+
+2. Configure clusterloader2 to generate KS performance workloads:
+
+   Starting from a local directory containing this KubeStellar git repo, run the following script based on your cluster environment:
+
+   a) cd into the `test/performance/common` directory from your local copy of the KubeStellar repo, for example:
+
+   ```bash 
+   cd .$HOME/kubestellar/test/performance/common
+   ```
+
+   b) set-up your environment:
+
+   ```bash 
+   ./setup-clusterloader2.sh --exp l
+   ```
+
+3. Configure the parameters of your workload:  
+
+   a) cd into the load configuration directory
+
+   ```bash 
+   cd  $CL2_DIR/testing/load/
+   ```
+  
+   b) configure the parameters to create the workload traffic (e.g., RandomizedLoad, SteppedLoad, Sequence, etc.)
+   
+   ```bash 
+   vi long-duration-exp-config.yaml
+   ``` 
+
+   More specifically, configure the following parameter: 
+
+   - numberOfWorkloadObjects: total number of objects to be created in the experiment `perf-test` namespace (*default value: "10"*)
+  
+   To learn more about clusterloader2 tuningSet, see the following: [tuningSets](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/docs/design.md#tuning-set) and [configurations](https://github.com/kubernetes/perf-tests/blob/fac2a5eec96fab76a4bc4858795df4544b729b0b/clusterloader2/api/types.go#L249).
+
+
+4. Deploy your first workload:
+
+   First, use a kubeconfig that includes contexts for WDS spaces and set its current context to a target WDS space (e.g., `wds1`):
+
+   ```bash 
+   kubectl config use-context wds1
+   ```
+
+   Output:
+   ```console 
+   Switched to context "wds1".
+   ```
+   
+   Optionally, check the kubeconfig contexts: 
+   ```bash 
+   kubectl config get-contexts
+   ```
+   
+   Output:
+   ```console 
+   CURRENT   NAME            CLUSTER         AUTHINFO        NAMESPACE
+             cluster1        kind-cluster1   kind-cluster1   
+             cluster2        kind-cluster2   kind-cluster2   
+             its1            its1-cluster    its1-admin      
+             kind-kubeflex   kind-kubeflex   kind-kubeflex   
+   *         wds1            wds1-cluster    wds1-admin      default
+             wds2            wds2-cluster    wds2-admin      default
+   ```
+
+   Second, create the binding Policy and namespace for your experiment using the following commands:
+
+   ```bash 
+   cd $CL2_DIR/testing/load/
+   kubectl apply -f long-duration-exp-setup.yaml
+   ```
+
+   Lastly, in the following set ``--kubeconfig`` flag to the path of the kubeconfig file used above and use the newly added Kubestellar provider (`--provider=ks`) and run the following command to create the workload:
+
+   ```bash
+   cd $CL2_DIR
+   go run cmd/clusterloader.go --testconfig=./testing/load/long-duration-exp-config.yaml --kubeconfig=<path>/wds-kubeconfig --provider=ks --v=2
```

Comment:
This command does not return quickly. The reader wonders: is it going to run forever, and I am expected to do the next step in a separate shell, or do I wait for this to return? If the latter, the reader should be given some idea of how long to wait.

In testing, I found that this command ran for about 611 seconds.

Reply:
I just added an equation at step 3-b to help the user to estimate the time duration of the execution of that command. I also updated the writing to make it clear that the reader can proceed with the next steps in parallel in a new shell.